README.md

```diff
@@ -2,51 +2,32 @@
 
 [![ipfs](http://img.shields.io/irc/%23ipfs.png)](https://webchat.freenode.net/?channels=ipfs)
 
-IPFS is a distributed file system that seeks to connect all computing devices with the same system of files. In some ways, this is similar to the original aims of the Web, but IPFS is actually more similar to a single bittorrent swarm exchanging git objects.
+Welcome to IPFS (the InterPlanetary File System)! Why not [watch a video](https://www.youtube.com/watch?v=8CMxDNuuAiQ) to get started?
 
-IPFS could become a new major subsystem of the internet. If built right, it could complement or replace HTTP. It could complement or replace even more. It sounds crazy. It _is_ crazy.
-
-#### IPFS IS A WORK IN PROGRESS
-
-Please note that IPFS is a work in progress. It is an ambitious plan to make the internet more free, open, secure, and high performance. It builds on the good ideas of numerous battle-tested distributed systems.
-
-[![Ready](https://badge.waffle.io/ipfs/ipfs.png?label=ready&title=Backlog)](https://waffle.io/ipfs/ipfs)
-
-## Security Issues
```

Comment:
we put security issues at the top because it is critical to drive attention. Maybe add it to the TOC as a heading like:

> Security Issues - Warnings and Reports